auth0_flutter/example/lib/main.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:async';
 
 import 'package:flutter/material.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
+// ignore: depend_on_referenced_packages
```

Comment:
Why is this exception needed?

Reply:
The linter complains that the driver package is not in the app's dependencies (it's on dev_dependencies).

Reply:
We could move it into the dependencies I believe, because it's actually used in the app itself so it does feel like a dependency instead of a devDependency to some extend.